packages/cli/src/utils/build.js

```diff
@@ -468,43 +629,27 @@ const _buildFunc = async ({
     debug('\nWarning: Skipping Validation');
   }
 
-  if (printProgress) {
-    startSpinner('Zipping project and dependencies');
-  }
-  await makeZip(tmpDir, path.join(wdir, zipPath), disableDependencyDetection);
+  maybeStartSpinner('Zipping project and dependencies');
+
+  const zipPath = path.join(appDir, BUILD_PATH);
+  await makeBuildZip(workingDir, zipPath, disableDependencyDetection);
   await makeSourceZip(
-    tmpDir,
-    path.join(wdir, sourceZipPath),
+    workingDir,
+    path.join(appDir, SOURCE_PATH),
     disableDependencyDetection,
   );
 
-  if (printProgress) {
-    endSpinner();
-    startSpinner('Testing build');
-  }
+  maybeEndSpinner();
 
-  if (!isWindows()) {
-    // TODO err, what should we do on windows?
-
-    // tries to do a reproducible build at least
-    // https://content.pivotal.io/blog/barriers-to-deterministic-reproducible-zip-files
-    // https://reproducible-builds.org/tools/ or strip-nondeterminism
-    await runCommand(
-      'find',
-      ['.', '-exec', 'touch', '-t', '201601010000', '{}', '+'],
-      { cwd: tmpDir },
-    );
-  }
```

Comment:
The old code did this _after_ building the zip, which didn't make sense. If you wanted to build a reproducible zip, you should do this _before_ creating the zip, right? I think this code has been redundant all this time. I've removed it from the new code.